pipeline/backend/dummy/dummy_test.go

```diff
@@ -12,9 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build test
```

Comment:
could you also re-add the tags here please?

Reply:
Same happens as mocks are excluded since #3831